### ISSUE TITLE
JsMin errors minifying jquery

### ISSUE THREAD

**Issue by dazinator** [Shazwazza/Smidge#22]
When attempting to use the `JsMin` preprocessor against `jquery` (latest version) it gives this error:

```
An unhandled exception occurred while processing the request.

Exception: Error: JSMIN Unterminated set in Regular Expression literal: 47
Smidge.FileProcessors.JsMin.action(Int32 d) in JsMin.cs, line 257

```

**Comment by Shazwazza** [comment]
I've updated that too recently , are you able to test with the latest smidge codebase? You can just run the build script to get the output or see appveyor for the latest build

**Comment by dazinator** [comment]
So, i just cloned from master, and referenced smidge directly - I get the same error:

```
Exception: Error: JSMIN Unterminated set in Regular Expression literal: 47
Smidge.FileProcessors.JsMin.action(Int32 d) in JsMin.cs, line 257
```

That's just trying to minify standard jquery library.

**Comment by Shazwazza** [comment]
This is fixed in rev: 903dbf7f59f503adcc56416284e62aba90899bfc with an updated jsmin

**Comment by dazinator** [comment]
Brilliant, will give it a try, cheers
